html5/_main_toc.html.slim

```diff
@@ -2,7 +2,8 @@ nav.sidebar-nav
   ul.nav.navbar-nav
     li
       .icon-title
-        a.bd-toc-link.main-link role="button" Getting Started
```

Comment:
Verified that on preview pages there is no issue with the new spacing.

Reply:
Yep, from what I am seeing this works fine.